gtas-parent/gtas-job-scheduler-war/src/main/java/gov/gtas/job/scheduler/Utils.java

```
@@ -0,0 +1,118 @@
+      //   targetPath = targetPath.resolveSibling(uniqueFilename);
+      //   logger.warn("Duplicate file in target dir! File has been renamed: " + targetPath.toString());
+      // }
+
+      Files.move(file.toPath(), targetPath, StandardCopyOption.ATOMIC_MOVE);
+    }
+    catch (Exception ex) {
+      throw new LoaderException("Could not move file to target dir: " + targetPath.toString());
+    }
+
+    return targetPath.toFile();
+  }
+
+  public static File writeToDisk(String fileName, String fileText, String target) {
+    if (!exists(fileName) || !exists(target)) {
+      return null;
+    }
+
+    Path targetPath = Paths.get(target + File.separator + fileName);
+    File f = null;
+
+    try {
+      // if(Files.exists(targetPath)) {
+      //   String uniqueFilename = getUniqueFilename(fileName);
+      //   targetPath = targetPath.resolveSibling(uniqueFilename);
+      // }
+
+      f = targetPath.toFile();
+
+      FileWriter fw = new FileWriter(f, false);
+      fw.write(fileText);
+      fw.close();
+    } catch (IOException e) {
+      // attempt to write it to an error directory here??
+      logger.error("error writing to directory " + target, e);
+    }
+    
+    return f;
+  }
+
+  private static String getUniqueFilename(String filename) {
+    String ext = FilenameUtils.getExtension(filename).isEmpty() ? "" : "." + FilenameUtils.getExtension(filename);
+    String nameOnly = FilenameUtils.getBaseName(filename);
+
+    return nameOnly + getTimestamp() + ext;
+  }
+
+  private static String getTimestamp() {
+    return " ~" + new Date().getTime();
+  }
+
+  public static String getText(String filePath) {
+    String text = null;
+    if (!exists(filePath)) return text;
+
+    try {
+      byte[] raw = FileUtils.readSmallFile(filePath);
+      String tmp = new String(raw, StandardCharsets.UTF_8);
+      text = ParseUtils.stripStxEtxHeaderAndFooter(tmp);
+      }
+      catch (IOException ex) {
+        //??
```

even if we eat and ignore an exception we could set the exception name to 'ignored' and log it.